webapp/channels/src/components/external_login_button/external_login_button.tsx

```diff
@@ -22,12 +25,17 @@ const ExternalLoginButton = ({
     label,
     style,
     direction = 'row',
+    onFocus,
+    onBlur
 }: ExternalLoginButtonType) => (
     <a
         id={id}
         className={classNames('external-login-button', {'direction-column': direction === 'column'}, id)}
         href={url}
         style={style}
+        onFocus={onFocus}
+        onBlur={onBlur}
+
```

Comment:
unnecessary blank line, please remove

Reply:
not sure why is not getting fixed, it not a blocker but is a good practice to not add unnecessary lines, specially before closing tags.